DESCRIPTION

```diff
@@ -1,6 +1,6 @@
-Package: xflow
+Package: recodeflow
 Type: Package
-Title: Converting xFlow Documents to PMML
+Title: Recode documents to PMML
```

Comment:
I'm not sure how we should rephrase this. Or should we just use the same phrasing?
`Converting recodeflow Documents to PMML`?

same question for the first line in the `R/pmml.R` file.
What do you think?